packages/material-renderers/src/controls/MaterialBooleanControl.tsx

```diff
@@ -46,30 +48,64 @@ export const MaterialBooleanControl = ({
   handleChange,
   errors,
   path,
-  config
+  config,
+  description
 }: ControlProps) => {
+
+  const isValid = errors.length === 0;
+  const appliedUiSchemaOptions = merge({}, config, uischema.options);
+
+  const showDescription = !isDescriptionHidden(
+    visible,
+    description,
+    false,
+    appliedUiSchemaOptions.showUnfocusedDescription
+  );
+
+  const showTooltip = !showDescription && !isDescriptionHidden(
```

Comment:
Maybe add a short comment why `true`, `true` are handed in as the two last parameters to make it easier to understand in the future